tdrs-frontend/src/components/Home/Home.test.jsx

```diff
@@ -302,6 +302,30 @@ describe('Pre-approval Home page', () => {
     expect(options.length).toEqual(4)
   })
 
+  it('should not show the stt combno box for non-STT users', () => {
```

Comment:
@jorgegonzalez is [this](https://github.com/raft-tech/TANF-app/blob/4a75b9d68fabb623ee42badc7da87a0ae9349ca6/tdrs-frontend/src/components/Home/Home.jsx#L170) check still happening? looks like i cant submit a request without completing the invisible fields on this form. 
![1640_raft](https://user-images.githubusercontent.com/63075587/158239978-e5093d28-d93b-4016-8837-5623333d94f8.PNG)

Reply:
Can you try again with the new changes?